instrumentation/google.golang.org/grpc/otelgrpc/interceptor.go

```diff
@@ -233,6 +233,15 @@ func (w *clientStream) sendStreamEvent(eventType streamEventType, err error) {
 	}
 }
 
+func target(cc *grpc.ClientConn) string {
+	// Since cc is ClientConn instead of ClientConnInterface, it can be nil in some
+	// situations.
+	if cc == nil {
+		return "unknown"
```

Comment:
Why `unknown`? This will have an impact on the produced telemetry. I guess that it will set improper peer attributes. See: https://github.com/open-telemetry/opentelemetry-go-contrib/blob/ee8b97343ce4e86ee0025de77e897aa94a22f974/instrumentation/google.golang.org/grpc/otelgrpc/interceptor.go#L466